### DISCUSSION TITLE
What is the api for output to problem terminal

### DISCUSSION THREAD

**Discussion by wss29** [eclipse-theia/theia#14035] [Q&A]
Hi,
   there is a problem in terminal in theia, I want to output some message to it, what is the API to do the output
![image](https://github.com/user-attachments/assets/297a4203-02ef-48fa-8add-86bc443c671a)

**Comment by tortmayr** [comment]
Hi @wss29,
the problems view is contributed by the `@theia/markers` extension.
The central component to manage problem markers is the [`ProblemManager`](https://github.com/eclipse-theia/theia/blob/master/packages/markers/src/browser/problem/problem-manager.ts).
You can inject this service to create new problem markers and modify existing ones.
